runtime/src/bank.rs

```diff
@@ -8054,6 +8056,24 @@ impl Bank {
         if new_feature_activations.contains(&feature_set::update_hashes_per_tick::id()) {
             self.apply_updated_hashes_per_tick(DEFAULT_HASHES_PER_TICK);
         }
+
+        if new_feature_activations.contains(&feature_set::programify_feature_gate_program::id()) {
+            let datapoint_name = "bank-progamify_feature_gate_program";
+            if let Err(e) = replace_account::replace_empty_account_with_upgradeable_program(
+                self,
+                &feature::id(),
+                &inline_feature_gate_program::noop_program::id(),
```

Comment:
Are these arguments swapped? My impression is that this code is trying to copy code from somewhere into the account with id of `feature::id()` but this code uses `feature_id()` as the `source` instead of the `destination`.

Reply:
They do appear to be swapped. Great catch, thanks! 😅

#33894